### DISCUSSION TITLE
Bugs? Cannot install my own SDK version device such as Android 10

### DISCUSSION THREAD

**Discussion by Kinferiority** [Genymobile/feedback#163] [Genymotion Desktop] [answered]
The function of the software didn't work.
I want to install Android 10 version by my own SDK tool, but even I choose the way to change the path.When creating the device,I cannot find the version I download.

<img width="1020" alt="image" src="https://github.com/user-attachments/assets/df77d164-509a-4470-97cd-8df3440df277">
<img width="969" alt="image" src="https://github.com/user-attachments/assets/a5ec814c-9e8b-406f-bb87-1c814b8cf844">
<img width="994" alt="image" src="https://github.com/user-attachments/assets/401910cf-be56-4135-9644-1db2e81b9e3f">

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @Kinferiority,

This option only allows to use ADB from a third party Android SDK; it does not includes third party images provided by Android Studio.

You can only use the Android images that we provide with Genymotion.
This is not a bug.

Thank you.
Best regards,
